programs/lfx-mentorship/2023/02-Jun-Aug/project_ideas.md

```diff
@@ -17,6 +17,14 @@
 
 ## Proposed Project ideas
 
+* [Armada](#armada)
+* [CoreDNS](#coredns)
+* [Jaeger](#jaeger)
+* [Knative](#knative)
+* [Tetragon](#tetragon)
+* [WasmEdge](#wasmedge)
+* [Konveyor](#konveyor)
+
```

Comment:
While I would appreciate the addition of toc, last GSoC term showed us that necessity of maintaining the toc causes a confusion for people when they are adding their ideas here.

Given that it is the last day of idea proposals, we can still keep it though. I would leave it to @nate-double-u